code/core/src/core-server/utils/StoryIndexGenerator.ts

```diff
@@ -368,9 +373,24 @@ export class StoryIndexGenerator {
     const entries: ((StoryIndexEntryWithExtra | DocsCacheEntry) & { tags: Tag[] })[] =
       indexInputs.map((input) => {
         const name = input.name ?? storyNameFromExport(input.exportName);
-        const componentPath =
-          input.rawComponentPath &&
-          this.resolveComponentPath(input.rawComponentPath, absolutePath, matchPath);
+        let componentPath;
+        if (input.rawComponentPath?.startsWith('#')) {
```

Comment:
In Storybook, we currently tell people to use a hash for import path mappings. But can this change? Is it theoretically allowed to use another prefix to make module mocking work?

cc @kasperpeulen

Reply:
AFAIK the spec says it has to be `#` symbol.